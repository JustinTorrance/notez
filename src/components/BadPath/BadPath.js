import React from 'react';

const BadPath = () => {
  return (
    <div className='bad-path-div'>
      <h2 className='bad-path-404'>404</h2>
      <p className='does-not-exist'>
        This page does not exist
      </p>
    </div>
  )
}

export default BadPath;