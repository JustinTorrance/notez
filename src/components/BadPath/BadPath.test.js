import React from 'react';
import { shallow } from 'enzyme';
import BadPath from './BadPath';

describe('BadPath', () => {
  const wrapper = shallow(<BadPath />);

  it('should match the snapshot', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});