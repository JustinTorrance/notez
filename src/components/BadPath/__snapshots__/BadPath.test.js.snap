// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BadPath should match the snapshot 1`] = `
"<div>
  <h2>
    404
  </h2>
  <p>
    This page does not exist
  </p>
</div>"
`;
